lib/user-methods.js

```diff
@@ -348,6 +348,23 @@ const staticMethods = {
 
 
 const instanceMethods = {
+    logout() {
+        this._logout();
+        const url = url_parse(this.server);
+        url.set('pathname', '/auth/revoke');
+        const headers = {
+            Authorization: this.token
+        };
+        const options = {
+            method: 'POST',
```

Comment:
This should send the token in the body (the idea is that with a valid token, you could invalidate another token):

https://github.com/realm/realm-object-server-private/blob/master/src/services/AuthService.ts#L587